ParabolInc/action
# Retro card groups

Fixes #1729.

## Testing

- Run the "drag and drop" story under "Reflection Card Grouping".
- Drag cards onto each other
- Click groups to expand/collapse
- Edit group titles
- Drag cards out of groups to "ungroup" them
- Note that when you drag the second-to-last out of a group, the group vanishes

Of note: in the demo, I'm not keeping track of the visual ordering of cards with respect to each other or the groups. So often when you group and/or ungroup, the group pops up in a place you wouldn't expect. This is intentional, as I wasn't sure if that work was in scope or not. There's evidently broader work that we'll need to prioritize regarding the more abstract "grouping page" or "grouping phase" in order to "catch" cards that are ungrouped, handle visual ordering/object constancy, etc.
